README.md

```diff
@@ -156,6 +159,28 @@ $ dart run custom_lint
 
 If you are working on a Flutter project, run `flutter pub run custom_lint` instead.
 
+
+### Debugging your custom lint plugin using hot-reload & dart debugger
```

Comment:
You probable already saw this, but this explains how to do it on the command line. You need to add the --enable-vm-service flag to dart. This will enable hot-reload on file changes, but it will not connect you to the dart debugger, you would need to open up devtools to interact with the debugger. It is typically easier just to use the IDE to start the linter, than working on the command line if you want to debug with breakpoints.

Reply:
You mean that you opened the Flutter devtool and set breakpoints there?

Reply:
I thought you had a way of connecting vscode with the debugger directly from the IDE

Reply:
Just press the debug button over the main, as mentioned in the README, this will connect the vscode debugger.

Reply:
Ah, I missed that line. I only saw the one about using command line

Reply:
Thanks for the quick reply btw!